framework/lib/components/search-bar/search-bar.tsx

```diff
@@ -1,13 +1,13 @@
-import React, { forwardRef, useCallback, useState } from 'react'
+import React, { forwardRef, useMemo, useState } from 'react'
 import {
   AsyncSelect,
   GroupBase,
   InputActionMeta,
   SelectInstance,
 } from 'chakra-react-select'
 import { filter, identity, is, test, toLower } from 'ramda'
-import debounce from 'lodash.debounce'
```

Comment:
If we remove this dependency it should be removed from package.json as well. We don't use it anywhere else right?

Reply:
It is still used in "useSortedIcons"